gitbot/deployhook.py

```diff
@@ -193,6 +193,8 @@ def process_pull_request() -> tuple[str, int]:
 
         if pull_request["merged"]:
             return respond("Pull request is already merged.", status_code=200)
+    elif ENV == "staging":
+        return respond("We do not support this for staging.", status_code=200)
```

Comment:
A developer opens a PR on each repo but the branch does not exist on `getsentry-test-repo`.
This is why we get alert like this:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/44410/174625709-615bef2b-1671-460d-9749-15831e2ba9af.png">